src/components/AnimatedFormWrapper.tsx

```
@@ -0,0 +1,133 @@
+          "videoId",
+          "players",
+        ]);
+        break;
+      case 1:
+        const stats = getValues().sets.flatMap((field, setIndex) =>
+          field.matches.flatMap((match, matchIndex) =>
+            match.playerSessions.flatMap((_, sessionIndex) => [
+              `sets.${setIndex}.matches.${matchIndex}.playerSessions.${sessionIndex}.playerStats` as const,
+            ]),
+          ),
+        );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)